corehq/apps/api/odata/views.py

```diff
@@ -86,6 +95,44 @@ def get(self, request, domain, app_id):
         return add_odata_headers(HttpResponse(metadata, content_type='application/xml'))
 
 
+class ODataCaseServiceFromExportInstanceView(View):
+
+    urlname = 'odata_case_service_from_export_instance'
+
+    @method_decorator(basic_auth_or_try_api_key_auth)
```

Comment:
All this decorator does is confirm they have access to the domain - I think this needs a permissions check too.

Reply:
Technically we aren't exposing information that isn't available in other places in HQ, but for completeness I added a check and tests: 0ccd80c0027721b0876ac0d989f1d006669c0ce9

Reply:
Isn't this case data?  (I assume the metadata views are fine)

Reply:
The service doc just has a list of the odata config ids in the domain